src/main/java/ToDo.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Represents a Task called ToDo.
+ */
+public class ToDo extends Task {
+    ToDo(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isCompleted = false;
+
+    }
+    /**
+     * Return an empty String to indicate there is no date line/event date for this ToDo.
+     *
+     * @return An empty string.
+     */
+    public String getDate() { return ""; }
```

Comment:
Should this be split across multiple lines rather than condensed into a single line?